src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/leaked-passwords/leakedPasswordsData.tsx

```diff
@@ -133,24 +134,26 @@ export const findFirstUnresolvedBreach = (
 };
 
 export async function updatePasswordsBreachStatus(
-  email: string,
+  emails: string,
   id: number,
   resolvedDataClass: Array<HibpBreachDataTypes[keyof HibpBreachDataTypes]>,
 ) {
   try {
-    const data: BreachResolutionRequest = {
-      affectedEmail: email,
-      breachId: id,
-      resolutionsChecked: resolvedDataClass,
-    };
+    for (const email of emails) {
+      const data: BreachResolutionRequest = {
+        affectedEmail: email,
+        breachId: id,
+        resolutionsChecked: resolvedDataClass,
+      };
 
-    const res = await fetch("/api/v1/user/breaches", {
-      method: "PUT",
-      body: JSON.stringify(data),
-    });
+      const res = await fetch("/api/v1/user/breaches", {
+        method: "PUT",
+        body: JSON.stringify(data),
+      });
```

Comment:
Heads-up: by putting this in a for loop, these requests are fired off in serial, i.e. the next one only begins after the previous response has come in. It might be better to do an `await Promise.all` on `emails.map(/* ... */)`.